arabic/unpacked/common-identifiers.js

```diff
@@ -0,0 +1,54 @@
+MathJax.Hub.Config({
+  // Math variables and functions
+  Arabic: {
+    identifiersMap: {
+      // Sets operations, and other stuff
+      'A': 'أ',
```

Comment:
You should probably use `"\u0623"` rather than `'أ'`, since the latter depends on the page encoding, while the former doesn't.  Scripts are loaded using the encoding of the page (last time I checked), so this would require the page to be UTF-8, and we don't know what encoding may be used on the page.  So it is best to use explicit unicode code points via `\uXXXX` instead.